GameOfRunes/GameOfRunes/GameEngine/Components/MultiplierComponent.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  MultiplierComponent.swift
+//  GameOfRunes
+//
+//  Created by Andy on 25/3/20.
+//  Copyright © 2020 TeamHoWan. All rights reserved.
+//
+
+import SpriteKit
+import GameplayKit
+
+class MultiplierComponent: GKComponent, Component {
+    var type: ComponentType {
+        .multiplierComponent
+    }
+    
+    private var _multipler: Int = 1
+    
+    var multiplier: Int {
```

Comment:
Maybe this isn't necessary if no checks are performed for the setting of the new value? Can use _multiplier directly, unless you anticipate some future checks?